### ISSUE TITLE
CORE: Using tag values with spaces fails writing the point

### ISSUE THREAD

**Issue by ioreskovic** [fsanaulla/chronicler#101]
```Scala
val pointWithTagWithSpace = Point(
    measurementName,
    List(InfluxTag("tag_name", "tag with spaces")),
    List(IntField("int_field", 42)),
    System.currentTimeMillis()
)

influx.database(databaseName).writePoint(pointWithTagWithSpace).onComplete {
  case Success(wr) if wr.isSuccess => println("Successfully wrote ??? points")
  case y @ _ => println("Failed writing points because " + y)
}
```

Outputs this:

```log
Failed writing points because Success(WriteResult(400,false,Some(com.github.fsanaulla.chronicler.core.model.BadRequestException: unable to parse 'chroniclerTestMm1,tag_name=tag with spaces int_field=42i 1540752319459': invalid field format)))
```

When you remove spaces, things pass
```Scala
val pointWithTagWithoutSpace = Point(
    measurementName,
    List(InfluxTag("tag_name", "tag_without_spaces")),
    List(IntField("int_field", 24)),
    System.currentTimeMillis()
)

influx.database(databaseName).writePoint(pointWithTagWithoutSpace).onComplete {
  case Success(wr) if wr.isSuccess => println("Successfully wrote ??? points")
  case y @ _ => println("Failed writing points because " + y)
}
```

with output

```log
Successfully wrote ??? points
```

As per Influx line protocol spec, it we want tag value with spaces, we have to escape them:
```Scala
val pointWithTagWithSpaceHax = Point(
    measurementName,
    List(InfluxTag("tag_name", "tag\\ with\\ spaces")),
    List(IntField("int_field", 42)),
    System.currentTimeMillis()
)

influx.database(databaseName).writePoint(pointWithTagWithSpaceHax).onComplete {
  case Success(wr) if wr.isSuccess => println("Successfully wrote ??? points")
  case y @ _ => println("Failed writing points because " + y)
}
```

with output

```log
Successfully wrote ??? points
```

Would it be possible to improve line serialization as such to handle this case?

**Comment by fsanaulla** [comment]
@ioreskovic great point will fix it in a while.

**Comment by fsanaulla** [comment]
Solved in #102.
